From 4e24bb31bcc76d6d218f3056b4c24a109d367561 Mon Sep 17 00:00:00 2001
From: Florian Loitsch <florian@loitsch.com>
Date: Mon, 23 Dec 2013 16:25:45 +0100
Subject: [PATCH] Fix build on ARMv8 64bit.

I previously used the wrong variable (_ARCH64EL_ instead of
__ARCH64EL__).

BUG=33

Edited-by: Marcin Juszkiewicz <mjuszkiewicz@redhat.com>
- path changed
- Changelog part dropped
---
diff --git a/src/utils.h b/src/utils.h
index c76f77d..c0b8c31 100644
--- qtdeclarative-opensource-src-5.2.0-rc1.orig/src/3rdparty/double-conversion/utils.h
+++ qtdeclarative-opensource-src-5.2.0-rc1/src/3rdparty/double-conversion/utils.h
@@ -63,7 +63,7 @@
     defined(__sparc__) || defined(__sparc) || defined(__s390__) || \
     defined(__SH4__) || defined(__alpha__) || \
     defined(_MIPS_ARCH_MIPS32R2) || \
-    defined(_AARCH64EL_)
+    defined(__AARCH64EL__)
 #define DOUBLE_CONVERSION_CORRECT_DOUBLE_OPERATIONS 1
 #elif defined(_M_IX86) || defined(__i386__) || defined(__i386)
 #if defined(_WIN32)
-- 
1.8.4.2

